src/mqt/ionshuttler/multi_shuttler/circuit_parsing.py

```
@@ -0,0 +1,101 @@
+        line = raw_line.strip()
+        if not line or line.startswith(("//", "#")):
+            continue
+        if line.startswith(_HEADER_PREFIXES):
+            continue
+        yield line
+
+
+def _load_quantum_circuit(qasm_str: str) -> QuantumCircuit:
+    try:
+        return QuantumCircuit.from_qasm_str(qasm_str)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)